Build failures using openjdk7

This appears to be a travisci problem.  The build was working on the legacy infrastructure (BlueBox) using openjdk 1.7.0_75.  On the new infrastructure (GCE), it is failing with a buffer overflow when it starts the tests resulting in a dead JVM.
 
working - 
OpenJDK 1.7.0_75
https://travis-ci.org/p6spy/p6spy/jobs/63590259

Failing build - 
OpenJDK 1.7.0_91
https://travis-ci.org/p6spy/p6spy/jobs/117577618

For now, the openjdk7 build has been disabled.

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
*** buffer overflow detected ***: /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java terminated
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(__fortify_fail+0x37)[0x7f7a48947e57]
/lib/x86_64-linux-gnu/libc.so.6(+0x108d50)[0x7f7a48946d50]
/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnet.so(Java_java_net_Inet4AddressImpl_getLocalHostName+0x190)[0x7f7a3c5559a0]
[0x7f7a3d012d98]
======= Memory map: ========
00400000-00401000 r-xp 00000000 08:01 403517                             /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java
00600000-00601000 r--p 00000000 08:01 403517                             /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java
00601000-00602000 rw-p 00001000 08:01 403517                             /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java
01944000-01965000 rw-p 00000000 00:00 0                                  [heap]
80a00000-81f00000 rw-p 00000000 00:00 0 
81f00000-8b000000 rw-p 00000000 00:00 0 
8b000000-8fe00000 rw-p 00000000 00:00 0 
8fe00000-d9000000 rw-p 00000000 00:00 0 
d9000000-db700000 rw-p 00000000 00:00 0 
db700000-100000000 rw-p 00000000 00:00 0 
7f7a10000000-7f7a10021000 rw-p 00000000 00:00 0 
7f7a10021000-7f7a14000000 ---p 00000000 00:00 0 
7f7a1692f000-7f7a16936000 r-xp 00000000 08:01 2503                       /lib/x86_64-linux-gnu/libnss_dns-2.15.so
7f7a16936000-7f7a16b35000 ---p 00007000 08:01 2503                       /lib/x86_64-linux-gnu/libnss_dns-2.15.so
7f7a16b35000-7f7a16b36000 r--p 00006000 08:01 2503                       /lib/x86_64-linux-gnu/libnss_dns-2.15.so
7f7a16b36000-7f7a16b37000 rw-p 00007000 08:01 2503                       /lib/x86_64-linux-gnu/libnss_dns-2.15.so
7f7a16b37000-7f7a16b47000 r-xp 00000000 08:01 656184                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnio.so
7f7a16b47000-7f7a16d47000 ---p 00010000 08:01 656184                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnio.so
7f7a16d47000-7f7a16d48000 r--p 00010000 08:01 656184                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnio.so
7f7a16d48000-7f7a16d49000 rw-p 00011000 08:01 656184                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnio.so
7f7a16d49000-7f7a17038000 rw-p 00000000 00:00 0 
7f7a17038000-7f7a17074000 r-xp 00000000 08:01 3360                       /lib/x86_64-linux-gnu/libpcre.so.3.12.1
7f7a17074000-7f7a17273000 ---p 0003c000 08:01 3360                       /lib/x86_64-linux-gnu/libpcre.so.3.12.1
7f7a17273000-7f7a17274000 r--p 0003b000 08:01 3360                       /lib/x86_64-linux-gnu/libpcre.so.3.12.1
7f7a17274000-7f7a17275000 rw-p 0003c000 08:01 3360                       /lib/x86_64-linux-gnu/libpcre.so.3.12.1
7f7a17275000-7f7a1727c000 r-xp 00000000 08:01 45453                      /usr/lib/x86_64-linux-gnu/libffi.so.6.0.0
7f7a1727c000-7f7a1747b000 ---p 00007000 08:01 45453                      /usr/lib/x86_64-linux-gnu/libffi.so.6.0.0
7f7a1747b000-7f7a1747c000 r--p 00006000 08:01 45453                      /usr/lib/x86_64-linux-gnu/libffi.so.6.0.0
7f7a1747c000-7f7a1747d000 rw-p 00007000 08:01 45453                      /usr/lib/x86_64-linux-gnu/libffi.so.6.0.0
7f7a1747d000-7f7a17495000 r-xp 00000000 08:01 2883                       /lib/x86_64-linux-gnu/libresolv-2.15.so
7f7a17495000-7f7a17695000 ---p 00018000 08:01 2883                       /lib/x86_64-linux-gnu/libresolv-2.15.so
7f7a17695000-7f7a17696000 r--p 00018000 08:01 2883                       /lib/x86_64-linux-gnu/libresolv-2.15.so
7f7a17696000-7f7a17697000 rw-p 00019000 08:01 2883                       /lib/x86_64-linux-gnu/libresolv-2.15.so
7f7a17697000-7f7a17699000 rw-p 00000000 00:00 0 
7f7a17699000-7f7a176b6000 r-xp 00000000 08:01 3390                       /lib/x86_64-linux-gnu/libselinux.so.1
7f7a176b6000-7f7a178b5000 ---p 0001d000 08:01 3390                       /lib/x86_64-linux-gnu/libselinux.so.1
7f7a178b5000-7f7a178b6000 r--p 0001c000 08:01 3390                       /lib/x86_64-linux-gnu/libselinux.so.1
7f7a178b6000-7f7a178b7000 rw-p 0001d000 08:01 3390                       /lib/x86_64-linux-gnu/libselinux.so.1
7f7a178b7000-7f7a178b8000 rw-p 00000000 00:00 0 
7f7a178b8000-7f7a178bb000 r-xp 00000000 08:01 45440                      /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so.0.3200.4
7f7a178bb000-7f7a17aba000 ---p 00003000 08:01 45440                      /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so.0.3200.4
7f7a17aba000-7f7a17abb000 r--p 00002000 08:01 45440                      /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so.0.3200.4
7f7a17abb000-7f7a17abc000 rw-p 00003000 08:01 45440                      /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so.0.3200.4
7f7a17abc000-7f7a17bae000 r-xp 00000000 08:01 3258                       /lib/x86_64-linux-gnu/libglib-2.0.so.0.3200.4
7f7a17bae000-7f7a17dae000 ---p 000f2000 08:01 3258                       /lib/x86_64-linux-gnu/libglib-2.0.so.0.3200.4
7f7a17dae000-7f7a17daf000 r--p 000f2000 08:01 3258                       /lib/x86_64-linux-gnu/libglib-2.0.so.0.3200.4
7f7a17daf000-7f7a17db0000 rw-p 000f3000 08:01 3258                       /lib/x86_64-linux-gnu/libglib-2.0.so.0.3200.4
7f7a17db0000-7f7a17db1000 rw-p 00000000 00:00 0 
7f7a17db1000-7f7a17dfe000 r-xp 00000000 08:01 45462                      /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3200.4
7f7a17dfe000-7f7a17ffe000 ---p 0004d000 08:01 45462                      /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3200.4
7f7a17ffe000-7f7a17fff000 r--p 0004d000 08:01 45462                      /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3200.4
7f7a17fff000-7f7a18000000 rw-p 0004e000 08:01 45462                      /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3200.4
7f7a18000000-7f7a18108000 rw-p 00000000 00:00 0 
7f7a18108000-7f7a1c000000 ---p 00000000 00:00 0 
7f7a1c000000-7f7a1c021000 rw-p 00000000 00:00 0 
7f7a1c021000-7f7a20000000 ---p 00000000 00:00 0 
7f7a20000000-7f7a20021000 rw-p 00000000 00:00 0 
7f7a20021000-7f7a24000000 ---p 00000000 00:00 0 
7f7a24000000-7f7a24021000 rw-p 00000000 00:00 0 
7f7a24021000-7f7a28000000 ---p 00000000 00:00 0 
7f7a28000000-7f7a28021000 rw-p 00000000 00:00 0 
7f7a28021000-7f7a2c000000 ---p 00000000 00:00 0 
7f7a2c000000-7f7a2c021000 rw-p 00000000 00:00 0 
7f7a2c021000-7f7a30000000 ---p 00000000 00:00 0 
7f7a30000000-7f7a30021000 rw-p 00000000 00:00 0 
7f7a30021000-7f7a34000000 ---p 00000000 00:00 0 
7f7a34050000-7f7a38000000 rw-p 00000000 00:00 0 
7f7a38000000-7f7a38021000 rw-p 00000000 00:00 0 
7f7a38021000-7f7a3c000000 ---p 00000000 00:00 0 
7f7a3c1ff000-7f7a3c347000 r-xp 00000000 08:01 45452                      /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0.3200.4
7f7a3c347000-7f7a3c546000 ---p 00148000 08:01 45452                      /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0.3200.4
7f7a3c546000-7f7a3c54a000 r--p 00147000 08:01 45452                      /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0.3200.4
7f7a3c54a000-7f7a3c54c000 rw-p 0014b000 08:01 45452                      /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0.3200.4
7f7a3c54c000-7f7a3c54e000 rw-p 00000000 00:00 0 
7f7a3c54e000-7f7a3c565000 r-xp 00000000 08:01 656218                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnet.so
7f7a3c565000-7f7a3c765000 ---p 00017000 08:01 656218                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnet.so
7f7a3c765000-7f7a3c766000 r--p 00017000 08:01 656218                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnet.so
7f7a3c766000-7f7a3c767000 rw-p 00018000 08:01 656218                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnet.so
7f7a3c767000-7f7a3c76f000 r-xp 00000000 08:01 656192                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libmanagement.so
7f7a3c76f000-7f7a3c96e000 ---p 00008000 08:01 656192                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libmanagement.so
7f7a3c96e000-7f7a3c96f000 r--p 00007000 08:01 656192                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libmanagement.so
7f7a3c96f000-7f7a3c970000 rw-p 00008000 08:01 656192                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libmanagement.so
7f7a3c970000-7f7a3c973000 ---p 00000000 00:00 0 
7f7a3c973000-7f7a3ca71000 rw-p 00000000 00:00 0                          [stack:7989]
7f7a3ca71000-7f7a3ca74000 ---p 00000000 00:00 0 
7f7a3ca74000-7f7a3cb72000 rw-p 00000000 00:00 0                          [stack:7988]
7f7a3cb72000-7f7a3cb75000 ---p 00000000 00:00 0 
7f7a3cb75000-7f7a3cc73000 rw-p 00000000 00:00 0                          [stack:7987]
7f7a3cc73000-7f7a3cc76000 ---p 00000000 00:00 0 
7f7a3cc76000-7f7a3cd74000 rw-p 00000000 00:00 0                          [stack:7986]
7f7a3cd74000-7f7a3ceff000 r--p 00000000 08:01 2511                       /usr/lib/locale/locale-archive
7f7a3ceff000-7f7a3cf02000 ---p 00000000 00:00 0 
7f7a3cf02000-7f7a3d000000 rw-p 00000000 00:00 0                          [stack:7985]
7f7a3d000000-7f7a3d270000 rwxp 00000000 00:00 0 
7f7a3d270000-7f7a4018d000 rw-p 00000000 00:00 0 
7f7a4018d000-7f7a44000000 ---p 00000000 00:00 0 
7f7a440c8000-7f7a440cb000 ---p 00000000 00:00 0 
7f7a440cb000-7f7a441c9000 rw-p 00000000 00:00 0                          [stack:7984]
7f7a441c9000-7f7a441ca000 ---p 00000000 00:00 0 
7f7a441ca000-7f7a44c1c000 rw-p 00000000 00:00 0                          [stack:7983]
7f7a44c1c000-7f7a44dce000 r--s 01d2e000 08:01 403539                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/rt.jar
7f7a44dce000-7f7a452c2000 rw-p 00000000 00:00 0 
7f7a452c2000-7f7a452c3000 ---p 00000000 00:00 0 
7f7a452c3000-7f7a453c3000 rw-p 00000000 00:00 0                          [stack:7982]
7f7a453c3000-7f7a453c4000 ---p 00000000 00:00 0 
7f7a453c4000-7f7a454cf000 rw-p 00000000 00:00 0                          [stack:7981]
7f7a454cf000-7f7a45517000 rw-p 00000000 00:00 0 
7f7a45517000-7f7a4553e000 rw-p 00000000 00:00 0 
7f7a4553e000-7f7a45787000 rw-p 00000000 00:00 0 
7f7a45787000-7f7a45792000 rw-p 00000000 00:00 0 
7f7a45792000-7f7a457da000 rw-p 00000000 00:00 0 
7f7a457da000-7f7a45801000 rw-p 00000000 00:00 0 
7f7a45801000-7f7a45a4a000 rw-p 00000000 00:00 0 
7f7a45a4a000-7f7a45a5e000 rw-p 00000000 00:00 0 
7f7a45a5e000-7f7a45b82000 rw-p 00000000 00:00 0 
7f7a45b82000-7f7a45b83000 rw-p 00000000 00:00 0 
7f7a45b83000-7f7a45b8b000 r-xp 00000000 08:01 656188                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libzip.so
7f7a45b8b000-7f7a45d8a000 ---p 00008000 08:01 656188                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libzip.so
7f7a45d8a000-7f7a45d8b000 r--p 00007000 08:01 656188                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libzip.so
7f7a45d8b000-7f7a45d8c000 rw-p 00008000 08:01 656188                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libzip.so
7f7a45d8c000-7f7a45d98000 r-xp 00000000 08:01 4725                       /lib/x86_64-linux-gnu/libnss_files-2.15.so
7f7a45d98000-7f7a45f97000 ---p 0000c000 08:01 4725                       /lib/x86_64-linux-gnu/libnss_files-2.15.so
7f7a45f97000-7f7a45f98000 r--p 0000b000 08:01 4725                       /lib/x86_64-linux-gnu/libnss_files-2.15.so
7f7a45f98000-7f7a45f99000 rw-p 0000c000 08:01 4725                       /lib/x86_64-linux-gnu/libnss_files-2.15.so
7f7a45f99000-7f7a45fa3000 r-xp 00000000 08:01 2877                       /lib/x86_64-linux-gnu/libnss_nis-2.15.so
7f7a45fa3000-7f7a461a3000 ---p 0000a000 08:01 2877                       /lib/x86_64-linux-gnu/libnss_nis-2.15.so
7f7a461a3000-7f7a461a4000 r--p 0000a000 08:01 2877                       /lib/x86_64-linux-gnu/libnss_nis-2.15.so
7f7a461a4000-7f7a461a5000 rw-p 0000b000 08:01 2877                       /lib/x86_64-linux-gnu/libnss_nis-2.15.so
7f7a461a5000-7f7a461bc000 r-xp 00000000 08:01 3803                       /lib/x86_64-linux-gnu/libnsl-2.15.so
7f7a461bc000-7f7a463bb000 ---p 00017000 08:01 3803                       /lib/x86_64-linux-gnu/libnsl-2.15.so
7f7a463bb000-7f7a463bc000 r--p 00016000 08:01 3803                       /lib/x86_64-linux-gnu/libnsl-2.15.so
7f7a463bc000-7f7a463bd000 rw-p 00017000 08:01 3803                       /lib/x86_64-linux-gnu/libnsl-2.15.so
7f7a463bd000-7f7a463bf000 rw-p 00000000 00:00 0 
7f7a463bf000-7f7a463c7000 r-xp 00000000 08:01 4717                       /lib/x86_64-linux-gnu/libnss_compat-2.15.so
7f7a463c7000-7f7a465c6000 ---p 00008000 08:01 4717                       /lib/x86_64-linux-gnu/libnss_compat-2.15.so
7f7a465c6000-7f7a465c7000 r--p 00007000 08:01 4717                       /lib/x86_64-linux-gnu/libnss_compat-2.15.so
7f7a465c7000-7f7a465c8000 rw-p 00008000 08:01 4717                       /lib/x86_64-linux-gnu/libnss_compat-2.15.so
7f7a465c8000-7f7a465f1000 r-xp 00000000 08:01 656196                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libjava.so
7f7a465f1000-7f7a467f0000 ---p 00029000 08:01 656196                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libjava.so
7f7a467f0000-7f7a467f1000 r--p 00028000 08:01 656196                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libjava.so
7f7a467f1000-7f7a467f3000 rw-p 00029000 08:01 656196                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libjava.so
7f7a467f3000-7f7a46801000 r-xp 00000000 08:01 656213                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libverify.so
7f7a46801000-7f7a46a00000 ---p 0000e000 08:01 656213                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libverify.so
7f7a46a00000-7f7a46a02000 r--p 0000d000 08:01 656213                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libverify.so
7f7a46a02000-7f7a46a03000 rw-p 0000f000 08:01 656213                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libverify.so
7f7a46a03000-7f7a46a0a000 r-xp 00000000 08:01 2881                       /lib/x86_64-linux-gnu/librt-2.15.so
7f7a46a0a000-7f7a46c09000 ---p 00007000 08:01 2881                       /lib/x86_64-linux-gnu/librt-2.15.so
7f7a46c09000-7f7a46c0a000 r--p 00006000 08:01 2881                       /lib/x86_64-linux-gnu/librt-2.15.so
7f7a46c0a000-7f7a46c0b000 rw-p 00007000 08:01 2881                       /lib/x86_64-linux-gnu/librt-2.15.so
7f7a46c0b000-7f7a46c20000 r-xp 00000000 08:01 3387                       /lib/x86_64-linux-gnu/libgcc_s.so.1
7f7a46c20000-7f7a46e1f000 ---p 00015000 08:01 3387                       /lib/x86_64-linux-gnu/libgcc_s.so.1
7f7a46e1f000-7f7a46e20000 r--p 00014000 08:01 3387                       /lib/x86_64-linux-gnu/libgcc_s.so.1
7f7a46e20000-7f7a46e21000 rw-p 00015000 08:01 3387                       /lib/x86_64-linux-gnu/libgcc_s.so.1
7f7a46e21000-7f7a46f1c000 r-xp 00000000 08:01 4817                       /lib/x86_64-linux-gnu/libm-2.15.so
7f7a46f1c000-7f7a4711b000 ---p 000fb000 08:01 4817                       /lib/x86_64-linux-gnu/libm-2.15.so
7f7a4711b000-7f7a4711c000 r--p 000fa000 08:01 4817                       /lib/x86_64-linux-gnu/libm-2.15.so
7f7a4711c000-7f7a4711d000 rw-p 000fb000 08:01 4817                       /lib/x86_64-linux-gnu/libm-2.15.so
7f7a4711d000-7f7a471ff000 r-xp 00000000 08:01 45439                      /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.16
7f7a471ff000-7f7a473fe000 ---p 000e2000 08:01 45439                      /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.16
7f7a473fe000-7f7a47406000 r--p 000e1000 08:01 45439                      /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.16
7f7a47406000-7f7a47408000 rw-p 000e9000 08:01 45439                      /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.16
7f7a47408000-7f7a4741d000 rw-p 00000000 00:00 0 
7f7a4741d000-7f7a47f15000 r-xp 00000000 08:01 656204                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/server/libjvm.so
7f7a47f15000-7f7a48115000 ---p 00af8000 08:01 656204                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/server/libjvm.so
7f7a48115000-7f7a481b6000 r--p 00af8000 08:01 656204                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/server/libjvm.so
7f7a481b6000-7f7a481d9000 rw-p 00b99000 08:01 656204                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/server/libjvm.so
7f7a481d9000-7f7a48206000 rw-p 00000000 00:00 0 
7f7a48206000-7f7a4821c000 r-xp 00000000 08:01 3322                       /lib/x86_64-linux-gnu/libz.so.1.2.3.4
7f7a4821c000-7f7a4841b000 ---p 00016000 08:01 3322                       /lib/x86_64-linux-gnu/libz.so.1.2.3.4
7f7a4841b000-7f7a4841c000 r--p 00015000 08:01 3322                       /lib/x86_64-linux-gnu/libz.so.1.2.3.4
7f7a4841c000-7f7a4841d000 rw-p 00016000 08:01 3322                       /lib/x86_64-linux-gnu/libz.so.1.2.3.4
7f7a4841d000-7f7a48435000 r-xp 00000000 08:01 4372                       /lib/x86_64-linux-gnu/libpthread-2.15.so
7f7a48435000-7f7a48634000 ---p 00018000 08:01 4372                       /lib/x86_64-linux-gnu/libpthread-2.15.so
7f7a48634000-7f7a48635000 r--p 00017000 08:01 4372                       /lib/x86_64-linux-gnu/libpthread-2.15.so
7f7a48635000-7f7a48636000 rw-p 00018000 08:01 4372                       /lib/x86_64-linux-gnu/libpthread-2.15.so
7f7a48636000-7f7a4863a000 rw-p 00000000 00:00 0 
7f7a4863a000-7f7a4863c000 r-xp 00000000 08:01 2538                       /lib/x86_64-linux-gnu/libdl-2.15.so
7f7a4863c000-7f7a4883c000 ---p 00002000 08:01 2538                       /lib/x86_64-linux-gnu/libdl-2.15.so
7f7a4883c000-7f7a4883d000 r--p 00002000 08:01 2538                       /lib/x86_64-linux-gnu/libdl-2.15.so
7f7a4883d000-7f7a4883e000 rw-p 00003000 08:01 2538                       /lib/x86_64-linux-gnu/libdl-2.15.so
7f7a4883e000-7f7a489f2000 r-xp 00000000 08:01 3793                       /lib/x86_64-linux-gnu/libc-2.15.so
7f7a489f2000-7f7a48bf1000 ---p 001b4000 08:01 3793                       /lib/x86_64-linux-gnu/libc-2.15.so
7f7a48bf1000-7f7a48bf5000 r--p 001b3000 08:01 3793                       /lib/x86_64-linux-gnu/libc-2.15.so
7f7a48bf5000-7f7a48bf7000 rw-p 001b7000 08:01 3793                       /lib/x86_64-linux-gnu/libc-2.15.so
7f7a48bf7000-7f7a48bfc000 rw-p 00000000 00:00 0 
7f7a48bfc000-7f7a48c09000 r-xp 00000000 08:01 656215                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/jli/libjli.so
7f7a48c09000-7f7a48e08000 ---p 0000d000 08:01 656215                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/jli/libjli.so
7f7a48e08000-7f7a48e09000 r--p 0000c000 08:01 656215                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/jli/libjli.so
7f7a48e09000-7f7a48e0a000 rw-p 0000d000 08:01 656215                     /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/jli/libjli.so
7f7a48e0a000-7f7a48e2c000 r-xp 00000000 08:01 4638                       /lib/x86_64-linux-gnu/ld-2.15.so
7f7a48e53000-7f7a48e5d000 rw-p 00000000 00:00 0 
7f7a48e5d000-7f7a48f13000 rw-p 00000000 00:00 0 
7f7a48f13000-7f7a48f16000 ---p 00000000 00:00 0 
7f7a48f16000-7f7a49019000 rw-p 00000000 00:00 0                          [stack:7980]
7f7a4901f000-7f7a49020000 rw-p 00000000 00:00 0 
7f7a49020000-7f7a49028000 rw-s 00000000 08:01 526003                     /tmp/hsperfdata_travis/7977
7f7a49028000-7f7a49029000 rw-p 00000000 00:00 0 
7f7a49029000-7f7a4902a000 r--p 00000000 00:00 0 
7f7a4902a000-7f7a4902c000 rw-p 00000000 00:00 0 
7f7a4902c000-7f7a4902d000 r--p 00022000 08:01 4638                       /lib/x86_64-linux-gnu/ld-2.15.so
7f7a4902d000-7f7a4902f000 rw-p 00023000 08:01 4638                       /lib/x86_64-linux-gnu/ld-2.15.so
7ffdf526b000-7ffdf528c000 rw-p 00000000 00:00 0                          [stack]
7ffdf5370000-7ffdf5372000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]
Aborted (core dumped)

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 20.138 s
[INFO] Finished at: 2016-03-21T22:22:04+00:00
[INFO] Final Memory: 35M/230M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project p6spy: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /home/travis/build/p6spy/p6spy && /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.port=1234 -Dcom.sun.management.jmxremote.ssl=false -Doracle.jdbc.timezoneAsRegion=false -Ddb2.jcc.override.traceLevel=0 -jar /home/travis/build/p6spy/p6spy/target/surefire/surefirebooter7022273581379161669.jar /home/travis/build/p6spy/p6spy/target/surefire/surefire4407517801231860999tmp /home/travis/build/p6spy/p6spy/target/surefire/surefire_08594013984412611697tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

travis_time:end:15a368b8:start=1458598902381883673,finish=1458598924277830533,duration=21895946860
[0K
[31;1mThe command "mvn -B test -P travis --settings ~/.m2/p6spySettings.xml;" exited with 1.[0m

Done. Your build exited with 1.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,7 @@ language: java
 
 jdk:
   - oraclejdk7
-#  - openjdk7
+  - openjdk7
   - oraclejdk8
 
 env:
@@ -17,6 +17,7 @@ before_install:
  #- script/travis/before_install_db2.sh
  - script/travis/before_install_oracle.sh
  - groovy script/travis/generateMavenSettings.groovy
+ - sudo hostname "$(hostname | cut -c1-63)" && sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts | sudo tee /etc/hosts # https://github.com/travis-ci/travis-ci/issues/5227
 
 before_script:
  - script/travis/before_script_firebird.sh

